CHANGELOG.md

```diff
@@ -2,6 +2,39 @@
+- Improve var naming ([#8726](https://github.com/serverless/serverless/pull/8726)) ([7136850](https://github.com/serverless/serverless/commit/71368507e4df5b5582bafb3db9dc8a2e89a8ed8a)) ([Mariusz Nowak](https://github.com/medikoo))
+- Reconfigure onExitPromise setup ([#8726](https://github.com/serverless/serverless/pull/8726)) ([22a03ce](https://github.com/serverless/serverless/commit/22a03ce0d7b1581747b121f862d0818f04120958)) ([Mariusz Nowak](https://github.com/medikoo))
+- Refactor init part to async/await ([#8726](https://github.com/serverless/serverless/pull/8726)) ([763fc34](https://github.com/serverless/serverless/commit/763fc342f7d82a86db00cbccab2f847d943d3e9e)) ([Mariusz Nowak](https://github.com/medikoo))
+- Refactor Serverless.run to async ([#8749](https://github.com/serverless/serverless/pull/8749)) ([30015ea](https://github.com/serverless/serverless/commit/30015eafd2fb9d2e82d8f34ee8f10c1fb4e536a0)) ([Piotr Grzesik](https://github.com/pgrzesik))
+- Reorder init setup ([#8726](https://github.com/serverless/serverless/pull/8726)) ([7548e1d](https://github.com/serverless/serverless/commit/7548e1d847f70127384400c027de3b6386fde698)) ([Mariusz Nowak](https://github.com/medikoo))
+- Seclude `cli/resolve-local-serverless-path` util ([#8726](https://github.com/serverless/serverless/pull/8726)) ([9d78348](https://github.com/serverless/serverless/commit/9d783482895d82a1bfdb627c4cc0debb32123d56)) ([Mariusz Nowak](https://github.com/medikoo))
+- Seclude ensureExists util ([#8744](https://github.com/serverless/serverless/pull/8744)) ([c3f59e4](https://github.com/serverless/serverless/commit/c3f59e4d785145c2e1ba7c1324f3afedba482479)) ([Piotr Grzesik](https://github.com/pgrzesik))
+- Seclude ServerlessError into lib/serverless-error.js ([#8743](https://github.com/serverless/serverless/pull/8743)) ([87790e5](https://github.com/serverless/serverless/commit/87790e50bd9c178aefd4f2ad8793c9c56fb8eb49)) ([Mariusz Nowak](https://github.com/medikoo))
+- Typos in schema ([#8735](https://github.com/serverless/serverless/issues/8735)) ([2b7568a](https://github.com/serverless/serverless/commit/2b7568a960c88dda8ab2bbe1b6c8dd238fa78a51)) ([Dmitry Shirokov](https://github.com/runk))
+- **CLI:** Seclude version output functionality out of CLI class ([#8741](https://github.com/serverless/serverless/pull/8741)) ([b61621a](https://github.com/serverless/serverless/commit/b61621adebb7eb33fd080db3fff13d7e9a32d99b)) ([Mariusz Nowak](https://github.com/medikoo))
+- **Templates:** Fix typo in package.json for template aws-nodejs-typescript ([#8754](https://github.com/serverless/serverless/issues/8754)) ([37398d0](https://github.com/serverless/serverless/commit/37398d06c582b1676c2aaa32708cfd515baf65b9)) ([Alexandre de Boutray](https://github.com/aldebout))
```

Comment:
Let's put updates related to _Templates_ to individual `Templates` section (there's somewhat independent of Framework logic, so I would not mix it in)